tests/test_command.py

```diff
@@ -17,24 +15,24 @@ def ec():
         parse_json=True,
         observe=False,
         observe_duration=0,
-        err_callback=ec,
+        err_callback=error_callback,
     )
 
     assert command.method == "method"
     assert command.path == "path"
     assert command.parse_json is True
     assert command.observe is False
     assert command.observe_duration == 0
-    assert command.err_callback == ec
+    assert command.err_callback is error_callback
 
 
 def test_result():
     """Test callback process_result."""
 
-    def pr(value):
+    def process_result(value):
```

Comment:
```suggestion
    def process_result(value: int):
```

Maybe add type hint here to? Not very important though.

Reply:
I didn't type the tests completely as we don't check that currently.

Reply:
Ok!